### ISSUE TITLE
Loading animation or text

### ISSUE THREAD

**Issue by josueharos** [ratiw/vuetable-2#7]
What is the correct way to show a text or animination for the table

i have this code
      
JS
```
 new Vue({
	el: #app-content,

	data: {	               
           paginationComponent: 'vuetable-pagination',
           paginationInfoTemplate: '', 
           loading: true
       }, 
     methods: {
    	onLoaded: function () {    	
    		this.loading = false;
    	}
     })
```
HTML
```
<div>
<div  v-if="loading"><span class="fa fa-spinner fa-spin fa-3x"></span>Loading ...</div>
    <vuetable
        ref="vuetable"
        api-url="{{ $setup->api_url}}"
        :fields="{{json_encode($setup->columns())}}"
        pagination-path="{{$setup->pagination_path}}"
        @vuetable:pagination-data="{{ $setup->pagination_data }}"
        :css="{tableClass: '{{$setup->css['tableClass']}}'}"
        @vuetable:loaded="onLoaded"
   >
    </vuetable>
</div>
```

For some reason when i execute the sentence in the onLoaded method, the table data dissapears what am i doing wrong? this was working with vue1, but i dont know what changed that is preventing from working

**Comment by ratiw** [comment]
@josueharos Did you get any error in the console?

**Comment by josueharos** [comment]
None, 
i can log to console and still see the table, but any modification to a property in my root instance hides the table

**Comment by ratiw** [comment]
@josueharos Your code looks fine to me. If there's no error in the console, I really don't know what it is. But if you could put your code into GitHub, maybe I can have a look. However, it may take awhile as I'm quite busy and only able to answer short question.

**Comment by lmj0011** [comment]
@josueharos 

I assume you ended up getting this to work. As for me, your example worked with vuetable-2@1.6.0

**Comment by itelmenko** [comment]
@ratiw In Vuetable-2 code:

```
default () {
        return {
          tableClass: 'ui blue selectable celled stackable attached table',
          loadingClass: 'loading',
          ascendingIcon: 'blue chevron up icon',
          descendingIcon: 'blue chevron down icon',
          detailRowClass: 'vuetable-detail-row',
          handleIcon: 'grey sidebar icon',
        }
      }
```

But loadingClass is not in use. Is it bug?
I see vuetable-1 examples have loading animation out of box. But vuetable-2 have no it.

**Comment by SimplyCorey** [comment]
I don't believe loadingClass is in use as well.  I have searched the codebase and I am unable to find it.

**Comment by ratiw** [comment]
@itelmenko @SimplyCorey Thanks for pointing out. The `loadingClass` is actually left-over from the first version.

In the first version of Vuetable, the wrapper class is part of the template which wrap Vuetable and its related pagination components, so showing and hiding load animation can be integrated. 

In Vuetable-2, it is independent and wrapper class is no longer needed. The showing / hiding of load animation should be handled outside of Vuetable. In Semantic UI, this is quite easy as a matter adding and removing `loading` css class, but in Bootstrap, you would need to do that yourself or using other library.

**Comment by ijunaid8989** [comment]
have anyone sorted this out for bootstrap?
